horton/scripts/wpart.py

```diff
@@ -119,7 +119,8 @@ def wpart_slow_analysis(wpart, mol):
     operators = []
     for ipure in xrange(npure):
         for iatom in xrange(wpart.natom):
-            operators.append(wpart.cache.load('overlap_operators', iatom)['olp_%05i' % ipure])
+            operators.append(wpart.cache.load('overlap_operators', iatom)[
+                             'olp_%05i' % ipure])
 
```

Comment:
P.S. If this wrapping was done automatically by pycharm, you should be really careful with the automatical fixes. It would be unfortunate to make things worse that way.